mungegithub/mungers/submit-queue.go

```diff
@@ -1129,6 +1141,7 @@ func (sq *SubmitQueue) doGithubE2EAndMerge(obj *github.MungeObject) bool {
 
 		// Wait for the retest to start
 		sq.SetMergeStatus(obj, ghE2EWaitingStart)
+		atomic.AddInt32(&sq.prsTested, 1)
```

Comment:
Hm, I know I said that this was probably OK, but maybe we should wait for the tests to go pending at least, to know that it actually took affect? Up to you unless @eparis has a strong opinion one way or the other.